linetools/spectra/xspectrum1d.py

```diff
@@ -40,22 +41,14 @@ class Spectrum1D(object):
 # Child Class of specutils/Spectrum1D
 # Generated by JXP to add functionality before it gets ingested in the
 # specutils distribution
-class XSpectrum1D(Spectrum1D):
-    """ A Class describing a spectrum.
-
-    It inherits from the Spectrum1D class in specutils.
+class XSpectrum1D(object):
+    """ A Class containing 1D spectra and associated methods
 
     Parameters
     ----------
-    data : `~numpy.ndarray`
-        flux of the spectrum
-
-    wcs : `spectrum1d.wcs.specwcs.BaseSpectrum1DWCS`-subclass
-        transformation between pixel coordinates and "dispersion" coordinates
-        this carries the unit of the dispersion
-
-    unit : `~astropy.unit.Unit` or None, optional
-        unit of the flux, default=None
+    _data : `~numpy.ndarray`
+        Structured array containing all of the data
+        This can be a set of 1D spectra
 
     mask : `~numpy.ndarray`, optional
```

Comment:
Does the code still support masks (I don't think it does)? If not remove this comment.